src/Tests/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs

```diff
@@ -431,7 +431,7 @@ public void ItAcceptsNoLogoAsCliArguments()
             }
         }
 
-        [WindowsOnlyFact]
```

Comment:
Out of curiosity, is this one deprecated?

Reply:
No. I was trying to run tests for Windows and Linux. But there were were issues like sdk was using older version of code coverage libs. So removed linux tests.